Dockerfile

```diff
@@ -15,11 +15,11 @@ RUN mkdir -p /algo && mkdir -p /algo/configs
 
 WORKDIR /algo
 COPY requirements.txt .
+
 RUN apk --no-cache add ${BUILD_PACKAGES} && \
     python -m pip --no-cache-dir install -U pip && \
-    python -m pip --no-cache-dir install virtualenv && \
-    python -m virtualenv env && \
-    source env/bin/activate && \
+    #python -m pip --no-cache-dir install virtualenv && \
```

Comment:
probably needs to be enabled?

Reply:
Local and CI tests pass without `virtualenv`.

Reply:
Ah, yes, right, we agreed to run it without virtualenv. Could you then just remote those lines, please?